css/css-text/white-space/break-spaces-newline-012.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Text: 'white-space: break-spaces', 10 white spaces and 2 line feeds</title>
+
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+  <link rel="help" href="https://www.w3.org/TR/css-text-3/#white-space-property">
+  <link rel="match" href="../../reference/ref-filled-green-100px-square.xht">
+  <link rel="stylesheet" type="text/css" href="/fonts/ahem.css">
+
+  <meta content="" name="flags">
+  <meta content="This test checks that when 'white-space' is set to 'break-spaces', then line feeds (&amp;NewLine; in the code) are preserved, just like with 'white-space: pre' or with 'white-space: pre-wrap'.">
+
+  <style>
+  div
+    {
+      font-family: Ahem;
+      font-size: 25px;
+      line-height: 1;
+      width: 4em;
+    }
+
+  div#overlapped-red-reference
+    {
+      background-color: red;
+      height: 4em;
+    }
+
+  div#overlapping-green-test
+    {
+      background-color: green;
+      bottom: 4em;
+      color: red;
+      position: relative;
+      white-space: break-spaces;
+    }
+  </style>
+
+  <p>Test passes if there is a filled green square and <strong>no red</strong>.
+
+  <div id="overlapped-red-reference"></div>
+
+  <div id="overlapping-green-test"> &NewLine;         &NewLine;</div>
```

Comment:
Same confusion: the first `&NewLine;` has an effect, but the second doesn't seem to.

Reply:
I can rehabilitate this 012 test. Indeed the second line feed is not involved as the "0" character is already on the 4th (bottom) line box.